makefile

```diff
@@ -32,6 +32,12 @@ lint:
 	@echo "--> Running linter"
 	@golangci-lint run
 	@go mod verify
+.PHONY: lint
+
+format:
+	find . -name '*.go' -type f -not -path "*.git*" -not -name '*.pb.go' -not -name '*pb_test.go' | xargs gofmt -w -s
+	find . -name '*.go' -type f -not -path "*.git*"  -not -name '*.pb.go' -not -name '*pb_test.go' | xargs goimports -w -local github.com/tendermint/tm-db
```

Comment:
I have a preference to remove the -local flag here, especially since most of our import lists are very short in this repo. (Same comment as on iavl.)